src/epiCD_helpers.R

```diff
@@ -269,4 +270,4 @@ jurisDictionary <- list(
 )
 
 PROVINCE_LOOKUP <- data.frame(province = c("BC", "NWT", "NL", "PEI"), 
-                              full_name = c("British Columbia", "Northwest Territories", "Newfoundland and Labrador", "Prince Edward Island"))
\ No newline at end of file
+                              full_name = c("British Columbia", "Northwest Territories", "Newfoundland and Labrador", "Prince Edward Island"))
```

Comment:
unsure what happened here, maybe just a encoding issue due to the fact that Im committing directly from GitHub